### PR TITLE
Expose the test api for mocking

### PR DESCRIPTION
**Pull request by ronco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,31 +62,75 @@ Check out [location-aware-ember-server](https://github.com/blimmer/location-awar
...
-* a [Promise](https://github.com/petkaantonov/bluebird/blob/master/API.md#core)  
-   when resolved, it returns the requested `index.html` string  
+* a [Promise](https://github.com/petkaantonov/bluebird/blob/master/API.md#core)
+   when resolved, it returns the requested `index.html` string
    when failed, it returns an [EmberCliDeployError](https://github.com/blimmer/node-ember-cli-deploy-redis/blob/develop/errors/ember-cli-deploy-error.js).
 
 
 ### options
-* `revisionQueryParam` (defaults to `index_key`)  
+* `revisionQueryParam` (defaults to `index_key`)
```
Comment (blimmer) [inline README.md]: This whitespace change messed up the formatting. Two spaces at the end of a line in markdown tells it not to continue wrapping text and to insert a break.

File: test/helpers/test-api.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+var Bluebird = require('bluebird');
+
+var redisClientApi = {
```
Comment (blimmer) [inline test/helpers/test-api.js]: It might make this more readable if this was a `ThenRedisClientApi`.

Comment (ronco) [inline test/helpers/test-api.js]: Do you want it exported as such? or just internally?

File: README.md (hunk 1 above)
Comment (ronco) [inline README.md]: I did not know this

File: test/helpers/test-api.js (hunk 2 above)
Comment (blimmer) [inline test/helpers/test-api.js]: I think it would be better to be exposed that way.

**Comment by blimmer** [comment]
Looks good to me! Thanks for the PR.

**Pull request merged**